.github/workflows/backport.yaml

```diff
@@ -0,0 +1,17 @@
+name: backport
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  cherry_pick_job:
+    permissions:
+      pull-requests: write
+      contents: write
+    if: github.event.pull_request.merged == true
+    secrets: inherit
+    uses: trustification/release-tools/.github/workflows/cherry-pick.yaml@main
```

Comment:
I can still see the name `cherry-pick.yaml` here. Am I missing something?

Reply:
Sorry, I missed that.